### ISSUE TITLE
Can Ensemble-Pytorch integrate multiple different models? [Support heterogenous models to be aggregated]

### ISSUE THREAD

**Issue by timewaitsfor** [xuyxu/Ensemble-Pytorch#5]
RT

**Comment by AaronX121** [comment]
Hi @timewaitsfor, thanks for this feature request ;-)

It would be nice if heterogenous models can be jointly passed into the ensemble wrapper. I will pin this issue and work on it.

**Comment by AaronX121** [comment]
Hi @timewaitsfor, I would be very happy if you could provide some suggestions on the design pattern of `MixWrapper` as you suggested. 

I have been thinking that it may not be a good idea to design a class that takes heterogenous models as the input, and condut a training stage on all of them, because the configuration on learning rate, batch size, etc., could be quite diverse for different models to achieve good performance. If we cater to all of these configurations, the input to the `MixWrapper` may be quite complex.

Instead, how about we provide a wrapper for **trained** base estimators? In such case, we take all fitted base estimators along with the test loader as the input, and we can then evaluate the accuracy, or other callbacks of these base estimators on the test loader. As to the training stage, it is up to you to decide how to train each of them.

**Comment by xuyxu** [comment]
Hi @timewaitsfor. This feature request will be involved after the version 0.1 released. Thanks for bringing us this feature request!
